tests/integration/mci/test_empty_submission.py

```diff
@@ -23,11 +23,11 @@ def empty_submission(self, form_type_id, eq_id):
         post_data = {
             'action[start_questionnaire]': 'Start Questionnaire'
         }
-        resp = self.client.post('/questionnaire/' + eq_id + '/789/introduction', data=post_data, follow_redirects=False)
+        resp = self.client.post('/questionnaire/' + eq_id + '/0205/201604/789/introduction', data=post_data, follow_redirects=False)
```

Comment:
The base of the urls here are the same throughout this file, we could wrap it up as a var to save on line length :) 

This goes for the other test files changed too.